packages/tasit-contracts/src/TasitContracts.js

```diff
@@ -30,73 +42,65 @@ const local = {
     address: "0x773f11ed472aa43e4ebaa963bcfbbea5a10c1bbd",
     abi: landRegistryABI,
   },
+  Marketplace: {
+    address: "0x2b3f477b0b056e91b8a389a904f2b6e1a9c00266",
+    abi: marketplaceABI,
+  },
+  GnosisSafe: {
+    address: "0x241bf551c22a5e1d05d3d066f5689d084a570016",
+    abi: gnosisSafeABI,
+  },
+};
+
+const goerli = {
   MANAToken: {
-    address: "0x332bfb4d705d3ce37c4bf06141c7104984e91e79",
+    address: "0x0e86f209729bf54763789cdbca9e8b94f0fd5333",
     abi: manaTokenABI,
   },
-  Marketplace: {
-    address: "0x70960e803a2bbe90c7db34edfdc2d1e81ed46b79",
-    abi: marketplaceABI,
+  EstateRegistry: {
+    address: "0xb32939da0c44bf255b9810421a55be095f9bb3f4",
+    abi: estateRegistryABI,
   },
-  MyERC20Full: {
-    address: "0x37E1A58dD465D33263D00185D065Ee36DD34CDb4",
-    abi: myERC20FullABI,
+  LANDProxy: {
+    address: "0xb35420eea0bc8af4f8a74414d080ea45f1ebf7bb",
+    abi: landProxyABI,
   },
-  MyERC721Full: {
-    address: "0x0E86f209729bf54763789CDBcA9E8b94f0FD5333",
-    abi: myERC721FullABI,
+  LANDRegistry: {
+    address: "0x332bfb4d705d3ce37c4bf06141c7104984e91e79",
+    abi: landRegistryABI,
   },
-  SampleContract: {
-    address: "0x6C4A015797DDDd87866451914eCe1e8b19261931",
-    abi: sampleContractABI,
+  Marketplace: {
+    address: "0x289c42facf691946b64b4370361b1303f0a463ef",
+    abi: marketplaceABI,
   },
 };
 
-import decentralandAddresses from "../3rd-parties/decentraland/contracts/addresses.json";
-const { ropsten: decentralandRopstenAddresses } = decentralandAddresses;
-const {
-  EstateProxy: ROPSTEN_ESTATE_REGISTRY_PROXY_ADDRESS,
-  EstateRegistry: ROPSTEN_ESTATE_REGISTRY_ADDRESS,
-  LANDProxy: ROPSTEN_LAND_PROXY_ADDRESS,
-  LANDRegistry: ROPSTEN_LAND_REGISTRY_ADDRESS,
-  MANAToken: ROPSTEN_MANA_TOKEN_ADDRESS,
-  MarketplaceProxy: ROPSTEN_MARKETPLACE_PROXY_ADDRESS,
-  Marketplace: ROPSTEN_MARKETPLACE_ADDRESS,
-} = decentralandRopstenAddresses;
-
 const ropsten = {
-  EstateProxy: {
-    address: ROPSTEN_ESTATE_REGISTRY_PROXY_ADDRESS,
-    abi: estateRegistryABI,
-  },
   EstateRegistry: {
-    address: ROPSTEN_ESTATE_REGISTRY_ADDRESS,
+    address: "0xb32939da0c44bf255b9810421a55be095f9bb3f4",
     abi: estateRegistryABI,
   },
   LANDProxy: {
-    address: ROPSTEN_LAND_PROXY_ADDRESS,
+    address: "0xb35420eea0bc8af4f8a74414d080ea45f1ebf7bb",
     abi: landProxyABI,
   },
   LANDRegistry: {
-    address: ROPSTEN_LAND_REGISTRY_ADDRESS,
+    address: "0x332bfb4d705d3ce37c4bf06141c7104984e91e79",
     abi: landRegistryABI,
   },
   MANAToken: {
-    address: ROPSTEN_MANA_TOKEN_ADDRESS,
+    address: "0x0e86f209729bf54763789cdbca9e8b94f0fd5333",
     abi: manaTokenABI,
   },
   Marketplace: {
-    address: ROPSTEN_MARKETPLACE_ADDRESS,
-    abi: marketplaceABI,
-  },
-  MarketplaceProxy: {
```

Comment:
We don't need the marketplace proxy anymore? Why is that?

Reply:
Our marketplace migration script isn't using proxy. Should I do that to simulate the same env that real contracts?

Reply:
Hmmm. Good to know. No strong opinion. I'll leave this up to you. We can always create an issue for it too.

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/252

Reply:
Thanks. Resolving.